webapi/src/main/scala/org/knora/webapi/routing/v1/ResourcesRouteV1.scala

```diff
@@ -329,7 +329,8 @@ final case class ResourcesRouteV1()(
                                ZIO.serviceWithZIO[MessageRelay](
                                  _.ask[GetFileMetadataResponse](GetFileMetadataRequest(tempFile, userProfile))
                                )
-                           } yield Some(makeFileValue(filename, fileMetadataResponse, projectShortcode))
+                             fileValue <- makeFileValue(filename, fileMetadataResponse, projectShortcode)
+                           } yield Some(fileValue)
                          case None => ZIO.none
```

Comment:
Using `ZIO.foreach` code like the following can be simplified substentially:
```
someOption match {
   case Some(foo) => functionWhichReturnsAZIO(foo)
   case None => ZIO.none
```
 as it becomes
 ```
 ZIO.foreach(someOption){foo=> functionWhichReturnsAZIO(foo)}
 ```
 wdyt?

Reply:
done.

I absolutely agree, I wasn't going for too much beautification because I mainly wanted to cross off another file that's throwing... but now that it's done, I'm very happy about it :)